README.md

```diff
@@ -80,17 +80,47 @@ $ cd parsec
 $ RUST_LOG=info cargo run
 ```
 
-Parsec Client Libraries can now communicate with the service. For example using the Rust Test client,
+Parsec Client Libraries can now communicate with the service. For example using the Rust client,
```

Comment:
Uhm... Now that we moved the provider out of the constructor for the client and we advise users to go through `list_providers`, `list_opcodes` first... I feel like this example is becoming quite heavy... it went from like 10 lines to probably 40-50 if I do the discovery part too... What do? Link to the client documentation once that's out?

Reply:
That's probably the best thing to do! Even more so because the example we would put here would not be checked for compilation often and might not even work at times 😬